<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="Plot" name="Plot_4"/>
    </item>
    <item row="0" column="0">
     <widget class="Plot" name="Plot_1">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Plot" name="Plot_3"/>
    </item>
    <item row="0" column="1">
     <widget class="Plot" name="Plot_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionZoom"/>
   <addaction name="actionExportPlot"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>102</width>
     <height>127</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>298</width>
     <height>582</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QComboBox" name="comboBoxSim">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lockParamsBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Sandler DiMaggio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>MATERIAL PARAMETERS</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Young</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QwtCounter" name="young_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>29269.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3" rowspan="2">
           <widget class="QPushButton" name="identifyBtn">
            <property name="text">
             <string>Identify</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Poisson</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QwtCounter" name="poisson_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>1.000000000000000</double>
            </property>
            <property name="stepButton2">
             <number>10</number>
            </property>
            <property name="value">
             <double>0.203000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QwtCounter" name="A_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>616.669999999999959</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QwtCounter" name="D_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.018768000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QwtCounter" name="B_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.003689000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QwtCounter" name="R_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.919690000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QwtCounter" name="C_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>1000.000000000000000</double>
            </property>
            <property name="stepButton3">
             <number>1000</number>
            </property>
            <property name="value">
             <double>111.480000000000004</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>W</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QwtCounter" name="W_counter">
            <property name="numButtons">
             <number>1</number>
            </property>
            <property name="maxValue">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.006605000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBoxMed">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Start idx:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QwtCounter" name="start_idx_value">
           <property name="numButtons">
            <number>1</number>
           </property>
           <property name="basicstep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="maxValue">
            <double>99.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="start_idx_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>End idx:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QwtCounter" name="end_idx_value">
           <property name="numButtons">
            <number>1</number>
           </property>
           <property name="basicstep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="maxValue">
            <double>99.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="end_idx_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Elastic Trans:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QwtCounter" name="elastic_trans_idx_value">
           <property name="numButtons">
            <number>1</number>
           </property>
           <property name="basicstep" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="maxValue">
            <double>99.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="elastic_trans_idx_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runSimBtn">
         <property name="text">
          <string>Run Simulation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Show Failure Envelope</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>label_22</zorder>
    <zorder>checkBox</zorder>
    <zorder>groupBox</zorder>
   </widget>
  </widget>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="actionExportPlot">
   <property name="text">
    <string>exportPlot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
  <customwidget>
   <class>Plot</class>
   <extends>QwtPlot</extends>
   <header>plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
